doc/whats_new/v0.22.rst

```diff
@@ -4,10 +4,10 @@
 
 .. _changes_0_22_2:
 
-Version 0.22.2
-==============
+Version 0.22.2.post1
+====================
 
-**February 28 2020**
+**March 3 2020**
```

Comment:
Maybe add a line somewhere to explain that 0.22.2.post1 includes a packaging fix for the source distribution but that the content of the packages is otherwise identical to the content of the wheels with the 0.22.2 version (without the .post1 suffix).